latch_cli/services/init/example_nfcore/task.py

```diff
@@ -26,15 +26,15 @@ def run_nfcore_fetchngs(
     ]
 
     try:
-        # When using shell=True, we pass the entire command as a single string as
-        # opposed to a list since the shell will parse the string into a list
-        # using its own rules.
-        subprocess.run(" ".join(nfcore_cmd), shell=True, check=True)
+        subprocess.run(" ".join(nfcore_cmd), check=True, capture_output=True)
```

Comment:
```suggestion
        subprocess.run(nfcore_cmd, check=True, capture_output=True)
```
since ur not using `shell=True` anymore